content/09.response-to-reviewers.md

```diff
@@ -0,0 +1,183 @@
+
+**We have updated our introduction to make our contributions for this work more clear.**
+```diff
+```
+
+>•       What are the limitations of the present work?
+
+**One limitation of this work is that changepoint validation can be challenging.
+Due to this being the first time this work has been performed, there isn't an available gold set.
+Our solution to circumvent this problem is that we provide a website that allows users to investigate our changepoint list further.
+**
+
+## Reviewer 4
+
+>This is a well-written manuscript to examine semantic shift in open access biomedical preprints and pre-publication peer-reviewed text. The methods are novel and  are clearly described. The results are clearly presented. It is a pleasure and very easy for audience to follow the paper. It adds to scientific value of the relative research field. I suggest acceptance.
+
+**We appreciate the positive feedback from the reviewer.**
+
+## Reviewer 5
+
+>Dear authors,
+>It is a pleasure to review your manuscript.
+
+**We appreciate the positive feedback from the reviewer.**
+
+>My suggestion:
+>1. Thoroughly revise the manuscript.
+
+**We have thoroughly revised our manuscript based on the feedback that was given.**
+
+>2. Visit the Submission Guidelines and place your manuscript according to the journal's guidelines.
+>Preparing main manuscript text
+>Preparing illustrations and figures
+>Preparing tables
+>Preparing additional files
+
+**We have examined the journal guidelines and have made sure everything is correct.**
+
+>3. The figures must be well displayed, the text of the figures must be legible.
+
+**We updated our figures to meet the journal's requirements.**
+
+>4. The introduction has few quotes, it is very short.
+
+**We have extended our introduction to include a more thorough discussion on previous work and highlight our contrubtions/motivation.**
+
+>5. The methodology is not well detailed, there is no figure that represents the pipeline.
+
+**We have updated our methods section to include a graphical depiction of our data pipeline.**
+
+## Reviewer 6
+
+>Reviewer Comments
+>Manuscript Number: Not Mentioned
+>The topic is exciting and shows how the words are changing over time. However, the authors must consider the following comments to improve the manuscript's quality.
+>
+>1.      The authors can use a graphical representation of the proposed work.
+
+**We have updated our methods section to include a graphical depiction of our data pipeline.**
+
+>2.      The authors must explain how the methodology is unique.
+
+**Our work is the first example where year variability has been accounted for within a word2vec model.**
+
+>3.      The resolution of the Figures can be improved for better visibility.
+
+**We updated our figures to meet the journal's requirements.**
+
+>4.      The authors should discuss the computational complexity of the methods.
+
+**We refrain from including this information as it appears outside of our paper scope.**
```

Comment:
```suggestion
**As with work focused on the training of machine learning models, it is difficult to estimate this precisely. Because we add inter- and intra-year variability, we note that the distance caluclations scale with the square of the number of models used per year and the number of years examined (if all combinations are calculated, as they are for our manuscript to examine stability over time); however, more efficiency could be gained in subsequent work by only examining subsequent year pairs now that the statistic has been evaluated.**
```